integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/nextbest/ConversationStrategyFactory.java

```diff
@@ -0,0 +1,46 @@
+package no.difi.meldingsutveksling.nextbest;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+import no.difi.meldingsutveksling.ServiceIdentifier;
+import no.difi.meldingsutveksling.config.IntegrasjonspunktProperties;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+@Component
+public class ConversationStrategyFactory {
+
+    private IntegrasjonspunktProperties props;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "props" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1068)